nuxt/index.js

```diff
@@ -47,10 +47,15 @@ module.exports = function nuxtBootstrapVue(moduleOptions = {}) {
       this.options.css.unshift('bootstrap/dist/css/bootstrap.css')
     }
 
-    // Transpile src
+    // Transpile src/
     this.options.build.transpile.push('bootstrap-vue/src')
```

Comment:
Should we conditionally set this only if `usePretranspiled` evaluates to true?

Reply:
This is not causing everything to be transpiled. But explicitly says do transpile `node_modules/bootstrap-vue-src/` requests. If one directly imports something from `src`, this prevents transpile issues.

Reply:
OK cool